packages/apps/docs/src/components/Markdown/Heading/Heading3.tsx

```diff
@@ -2,6 +2,10 @@ import { IHeader, TaggedHeading } from './Heading';
 
 import React, { FC } from 'react';
 
-export const Heading3: FC<IHeader> = ({ children }) => {
-  return <TaggedHeading as="h3">{children}</TaggedHeading>;
+export const Heading3: FC<IHeader> = ({ children, ...rest }) => {
+  return (
+    <TaggedHeading as="h3" {...rest}>
```

Comment:
this rest is not needed. that was just for test, right?

Reply:
no! it's required then only `index` get passed to down the component. Else I could use only `index` explicitly